src/main/java/github/scarsz/discordsrv/listeners/DiscordChatListener.java

```diff
@@ -90,7 +100,7 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
         // enforce required account linking
         if (DiscordSRV.config().getBoolean("DiscordChatChannelRequireLinkedAccount")) {
             if (DiscordSRV.getPlugin().getAccountLinkManager() == null) {
-                event.getAuthor().openPrivateChannel().queue(privateChannel -> privateChannel.sendMessage(LangUtil.Message.FAILED_TO_CHECK_LINKED_ACCOUNT.toString()).queue());
+                event.getAuthor().openPrivateChannel().queue(privateChannel -> privateChannel.sendMessage(LangUtil.Message.FAILED_TO_CHECK_LINKED_ACCOUNT.toString()).queue(null, new ErrorHandler().ignore(ErrorResponse.CANNOT_SEND_TO_USER)));
```

Comment:
Sure ignoring this exception is fine, but that is not a good solution to avoiding error responses when sending **invalid requests** (event.getAuthor() is a fake user when the message is from a webhook)